### PR TITLE
Add UML diagrams for CdCommand and Internal File Explorer

### PR DESCRIPTION
**Pull request by Ma-Yueran**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -262,26 +267,29 @@ Then, a new `Tag` is created using the modified `Set<Label>` and added back to t
...
-**Implementation of Internal File Explorer:**
+Implementation of Internal File Explorer:
 
 Model
 
-The model class `CurrentPath` saves the current directory of the explorer, and keeps a `FileList` of the children files under 
+The model class `CurrentPath` saves the current directory of the explorer, and keeps a `FileList` that contains the children files under 
 that directory.
 
+This is the class diagram for CurrentPath.
```
Comment (luo-git) [inline docs/DeveloperGuide.md]: Need a `<br>` here.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -219,10 +219,15 @@ It then gets the filepath of the `Tag` before safely deleting it. Then, a new `T
 changes the current directory of the HelloFile internal File Explorer. `CommandException` is thrown if the given directory 
 is invalid, cannot be found, or cannot be set as the current directory (*e.g. the given directory is not a folder*).
 
-CdCommand calls `setAddress` in `CurrentPath` to set the current directory to the absolute address parsed from the user input.
-Then, `CurrentPath` will update the list of children files `FileList` to fit the new current directory.
-Moreover, it also updates `javafx.scene.control.ListView` in `FileExplorer` as `ListView` is bound to the `FileList` of the children files 
-under the `CurrentPath`.
+This sequence diagram shows a successful execution of `CdCommand`.
```
Comment (luo-git) [inline docs/DeveloperGuide.md]: Need a `<br>` here.

**Review by luo-git — CHANGES_REQUESTED** [review]
Looks good. Just 2 `<br>` to make the formatting nicer.

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (Ma-Yueran) [reply docs/DeveloperGuide.md]: Changed

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Ma-Yueran) [reply docs/DeveloperGuide.md]: Changed

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -219,10 +219,15 @@ It then gets the filepath of the `Tag` before safely deleting it. Then, a new `T
 changes the current directory of the HelloFile internal File Explorer. `CommandException` is thrown if the given directory 
 is invalid, cannot be found, or cannot be set as the current directory (*e.g. the given directory is not a folder*).
 
-CdCommand calls `setAddress` in `CurrentPath` to set the current directory to the absolute address parsed from the user input.
-Then, `CurrentPath` will update the list of children files `FileList` to fit the new current directory.
-Moreover, it also updates `javafx.scene.control.ListView` in `FileExplorer` as `ListView` is bound to the `FileList` of the children files 
-under the `CurrentPath`.
+This sequence diagram shows a successful execution of `CdCommand`.<br>
+![FindSequenceDiagram](images/CdCommandSuccessSequenceDiagram.png)
```
Comment (anthony6401) [inline docs/DeveloperGuide.md]: I think you forget to change the FindSequenceDiagram.

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -271,26 +276,29 @@ Then, a new `Tag` is created using the modified `Set<Label>` and added back to t
...
+Implementation of Internal File Explorer:
 
 Model
 
-The model class `CurrentPath` saves the current directory of the explorer, and keeps a `FileList` of the children files under 
+The model class `CurrentPath` saves the current directory of the explorer, and keeps a `FileList` that contains the children files under 
 that directory.
 
+This is the class diagram for CurrentPath.<br>
+![UnlabelSuccessSequence](images/CurrentPathClassDiagram.png)
```
Comment (anthony6401) [inline docs/DeveloperGuide.md]: In here too.

**Review by anthony6401 — COMMENTED** [review]
Looks good to merge, but I think there is some typos.

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (Ma-Yueran) [reply docs/DeveloperGuide.md]: Oooo, ya, thanks for pointing that out

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (Ma-Yueran) [reply docs/DeveloperGuide.md]: Forgot to change that after pasting

Comment (Ma-Yueran) [reply docs/DeveloperGuide.md]: Updated

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (Ma-Yueran) [reply docs/DeveloperGuide.md]: Updated

**Review by luo-git — APPROVED** [review]
LGTM!

**Pull request merged**